### ISSUE TITLE
Details about the variables

### ISSUE THREAD

**Issue by cOsprey** [iprapas/landslide-sar-unet#4]
The datacube has topophase_flat_imag, topophase_flat_real, topophase_cor_0, topophase_cor_1 can you explain or provide more details about them? What is meant by flat and cor_0 and cor_1?

**Comment by iprapas** [comment]
If i remember correctly, these are the insar phase calculations. Flat imag(inary), real should be the difference in complex form, and cor should have the coherence. Not sure about the difference between cor_0 and cor_1. @weiji14 could you enlighten us maybe?

**Comment by weiji14** [comment]
Yes, these are the interferometric products produced from ISCE2 (we had documented some details at https://gitlab.com/frontierdevelopmentlab/2022-us-sarchangedetection/deepslide/-/blob/main/notebooks/README.md#13-insar-processing-using-isce2-data_isce_processing). The original output from ISCE2 is a file called `topophase.cor.tif`, and I'm having trouble remembering what cor_0 and cor_1 means too (the [ISCE manual](https://github.com/isce-framework/isce2-docs/blob/master/manual/ISCE.pdf) isn't very helpful here) 😅. A quick search yielded this page - https://adenaheem.com/standard-procedure-for-generating-differential-dems-from-insar-using-isce-strimapapp/ which says:

> topophase.flat: flattened (geometrical phase removed) and multi-looked interferogram.(one band complex64 data).
> topophase.cor: coherence and magnitude for the flattened multi-looked interferogram. (two bands float32 data).

Hopefully this is a meaningful explanation?

**Comment by cOsprey** [comment]
@iprapas and @weiji14, thank you. I am confused about cor_0 and cor_1, as generally the coherence should have value between 0 and 1, which is correct for cor_1 but cor_0 has values more than 1, seems something else.
